bmc/core/fixed_point.h

```diff
@@ -3578,11 +3578,13 @@ fxp_t fxp_double_to_fxp(double value)
   {
     tmp = (fxp_t) ftemp;
     double residue = ftemp - tmp;
-    if((value < 0) && (residue != 0))
+    /*if((value < 0) && (residue != 0))
```

Comment:
Please remove the comments from your PR. Your commits already keep the history of the changes,